packages/examples/create-react-app/sdk-copy.sh

```diff
@@ -4,10 +4,10 @@
 reldir="$( dirname -- "$0"; )";
 cd "$reldir/../../..";
 SDK_WORKSPACE_DIR="$( pwd; )";
-COMM_LAYER_DIR="$SDK_WORKSPACE_DIR/packages/sdk-communication-layer/dist"
```

Comment:
why removing the hack?

Reply:
see: https://github.com/MetaMask/metamask-sdk/pull/1092/files#diff-86a379f8f6064ca100e36683ec5ef0802fc31797e48023ab83a16d9fe91d86b3R26-R27